ZenPacks/zenoss/Microsoft/Windows/modeler/WinRMPlugin.py

```diff
@@ -212,6 +235,8 @@ def collect(self, device, log):
             try:
                 query_results = yield client.do_collect(
                     conn_info, query_map.iterkeys())
+                msg = "connection for %s is established" % device.id
```

Comment:
The device id is redundant here too since it'll already be a field in the event.